techmatters/terraso-backend
# feat: allow mp4 media type uploads to story maps

## Description
Add a media validation check to handle_config_media to 

### Related PRs
https://github.com/techmatters/terraso-web-client/compare/main...feat/allow-mp4-uploads-to-storymaps

### Related Issues
https://github.com/techmatters/terraso-product/issues/145

### Verification steps
1. On homepage, click on "Create New Story Map"
2. Add a title and click on "Add new chapter"
3. Click on "Add Media" and upload a mp4 file
4. Click "Publish" and should be successful
